### DISCUSSION TITLE
Page Speed is not higher we expected

### DISCUSSION THREAD

**Discussion by brkdmn35** [vercel/next.js#26525] [Help]
Hello, we have been planning to transfer our c# mvc project to Next.js to use SSR for better SEO performance. But after completed our first page the page score was 20 for mobile 50 for desktop. 

To increase this score some optimisation added to page, Load some packages after scroll, preconnect, preload for 3rd party requests and dynamic import for components which is not on viewport when page loading. All data moved to page as static props until we figure out this speed problem. So we are sure the problem is not about backend requests.

With these optimisations page score increased to 50 for mobile 90 for desktop aproximetly.

But these optimisation like loading components after scroll or dynamic imports caused to loose data on page source code. So these codes are important for SEO, google bots searching pages on source code. If i remove these optimisation, page score will be decrease. Even we couldn't reach current score with these optimisations.

![image](https://user-images.githubusercontent.com/34067493/123104585-c0f14e80-d43f-11eb-8e84-17c9640ef676.png)

Next Link: https://next-staging.hesapkurdu.com/ihtiyac-kredisi

Current Page: https://www.hesapkurdu.com/ihtiyac-kredisi

But our current page had almost 100 score on desktop and 75 for mobile. How can we increase our performance to this level or is there any problem that we didn't noticed.

I would be really greatful for suggestion and advices.

Thank you 😄

**Comment by bit2world** [comment]
Hello, I am from freelancer.
I think you are right.
In my opinion, this just depends on speed.

**Reply by bit2world** [reply]
Using Next.js, you can implement two types of tech.
One is SSR.
The other one is SSG.
In your case, you have to use SSR.

**Reply by bit2world** [reply]
I can help you to build your site using Next.js.
As I see it, current site is not React or Vue.
Finally, I wanna work with you.

**Reply by brkdmn35** [reply]
Thank you for answer, as I mentioned before current site developed as c# mvc project. We already started to developed by using Next.js with SSR. I was trying to say my some optimisation on next.js break some SEO advantages. With these optimisation we increased page speed but I am wonder how can we increase page speed without these dynamic import, lazy load exc... The next link represents next.js page.

**Reply by brkdmn35** [reply]
Is it possible to make these page score 100 for desktop and 75 for mobile. I am not expert on next.js yet. So, there must be some problem I couldn't notice.

**Reply by bit2world** [reply]
Yes, it is possible.

**Reply by bit2world** [reply]
Nearly, 8x faster than React.

**Reply by bit2world** [reply]
Totally, about 10x faster than your earlier one.

**Reply by bit2world** [reply]
Let me know if you need Next.js help.
